.github/workflows/release.yaml

```diff
@@ -0,0 +1,113 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - main
+      - '[0-9]+.[0-9]+.x'
+  workflow_dispatch:
+
+env:
+  # Default minimum version of Go to support.
+  DEFAULT_GO_VERSION: 1.20
```

Comment:
this was set to 1.19 in the aws validator plugin

Reply:
may as well go all the way to 1.21 then... wdyt?